.github/workflows/podvm.yaml

```diff
@@ -40,6 +44,19 @@ jobs:
     steps:
     - name: Checkout Code
       uses: actions/checkout@v3
+      with:
+        fetch-depth: 0
+        ref: ${{ inputs.caa_src_ref }}
+
+    - name: Read properties from versions.yaml
+      run: |
+        # There reference to CAA sources will honored if passed to this
+        # workflow via inputs.
+        caa_src_ref="${{ inputs.caa_src_ref }}"
+        [ -n "$caa_src_ref" ] || \
+          caa_src_ref="$(yq '.git.cloud-api-adaptor.reference' versions.yaml)"
+        [ -n "$caa_src_ref" ]
+        echo "CAA_SRC_REF=${caa_src_ref}" >> $GITHUB_ENV
```

Comment:
The same question goes here, isn't it possible to forward the `CAA_SRC_REF` to the downstream steps?

Reply:
`echo "CAA_SRC_REF=${caa_src_ref}" >> $GITHUB_ENV` ensures `CAA_SRC_REF` is forwarded to downstream steps **on same job** . Do you mean between jobs?